<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FaceAttendance</class>
 <widget class="QMainWindow" name="FaceAttendance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FaceAttendance</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget_leftback" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>480</width>
      <height>480</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QWidget" name="widget_success" native="true">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>340</y>
       <width>251</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px;
border-radius: 12px;
background-color: rgba(49, 50, 54, 70);</string>
     </property>
     <widget class="QLabel" name="label_success">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>13</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Alimama ShuHeiTi</family>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>认证成功</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_face">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>480</width>
       <height>480</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label_face</zorder>
    <zorder>widget_success</zorder>
   </widget>
   <widget class="QWidget" name="widget_rightback" native="true">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>0</y>
      <width>320</width>
      <height>480</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(52, 55, 64);</string>
    </property>
    <widget class="QLabel" name="label_title">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>261</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Alimama ShuHeiTi</family>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>人脸识别考勤系统</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>160</y>
       <width>261</width>
       <height>291</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_id" native="true">
        <property name="styleSheet">
         <string notr="true">QWidget{
	border: 1px;
	border-radius: 12px;

	background-color: rgb(37, 40, 49);
}</string>
        </property>
        <widget class="QLabel" name="label_id">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>3</y>
           <width>31</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Alimama ShuHeiTi</family>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>ID</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_id">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>10</y>
           <width>141</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Alimama ShuHeiTi</family>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_name" native="true">
        <property name="styleSheet">
         <string notr="true">QWidget{
	border: 1px;
	border-radius: 12px;

	background-color: rgb(37, 40, 49);
}</string>
        </property>
        <widget class="QLabel" name="label_name">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>3</y>
           <width>71</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Alimama ShuHeiTi</family>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>NAME</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_name">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>10</y>
           <width>141</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Alimama ShuHeiTi</family>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_id_3" native="true">
        <property name="styleSheet">
         <string notr="true">QWidget{
	border: 1px;
	border-radius: 12px;

	background-color: rgb(37, 40, 49);
}</string>
        </property>
        <widget class="QLabel" name="label_sector">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>3</y>
           <width>91</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Alimama ShuHeiTi</family>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>SECTOR</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_sector">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>10</y>
           <width>131</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Alimama ShuHeiTi</family>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_time" native="true">
        <property name="styleSheet">
         <string notr="true">QWidget{
	border: 1px;
	border-radius: 12px;

	background-color: rgb(37, 40, 49);
}</string>
        </property>
        <widget class="QLabel" name="label_time">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>3</y>
           <width>61</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Alimama ShuHeiTi</family>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>TIME</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_time">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>10</y>
           <width>151</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Alimama ShuHeiTi</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
